lib/change-memory-protection.yml

```diff
@@ -5,7 +5,7 @@ rule:
       - "@mr-tz"
     lib: true
     scopes:
-      static: basic block
+      static: function
```

Comment:
can you explain while the scopes here are relaxed? is it so there can be a basic block sub scope?

Reply:
if so, i think maybe we can have sub scope blocks that are the same as the outer scope (i *think*). we should try.

Reply:
Yes, unfortunately this forces us to loosen the scopes of some of the rules and the rules that subsequently match them.

Reply:
> if so, i think maybe we can have sub scope blocks that are the same as the outer scope (i _think_). we should try.

I tried this at first, but capa outputs the following error: _ERROR    capa: invalid rule: `basic block` subscope supported only for `function` scope_

Reply:
i think we might be able to relax this. i'll try it tomorrow.

Reply:
see https://github.com/mandiant/capa/pull/2584. once we merge that, let's re-tighten the scopes in this PR.